ark_sdk_python/models/common/identity/ark_identity_directory_schemas.py

```diff
@@ -123,13 +124,13 @@ class UserResult(ArkModel):
 
 class UsersResult(ArkModel):
     results: List[UserResult] = Field(alias='Results')
-    full_count: Optional[int] = Field(alias='FullCount')
+    full_count: Optional[int] = Field(default=None, alias='FullCount')
 
 
 class QueryResult(ArkModel):
-    groups: Optional[GroupsResult] = Field(alias='Group')
-    roles: Optional[RolesResult] = Field()
-    users: Optional[UsersResult] = Field(alias='User')
+    groups: Optional[GroupsResult] = Field(default=None, alias='Group')
+    roles: Optional[RolesResult] = Field(None)
```

Comment:
Do a search replace on all
```suggestion
    roles: Optional[RolesResult] = Field(default=None)
```

Reply:
fixed